pdp/static/js/pdp_raster_map.js

```diff
@@ -53,25 +87,31 @@ function dasToUnitsSince(data) {
         m = reg.exec(s),
         units = m[1],
         dateString = m[3],
-        sDate;
-
+        sDate;   
+    
+    var calendar;
+    reg = /calendar \"(standard|365_day|360_day)\"/,
```

Comment:
Ensure that your code also supports the other valid aliases for standard and 365_day that are described [in section 4.4.1 of this document](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch04s04.html). In particular, make sure that you support `noleap` and `gregorian`. I don't think that we need to support `all_leap`, `proleptic_gregorian`, or `julian` at this time.